test/aerospike_clj/client_test.clj

```diff
@@ -167,7 +228,7 @@
 (deftest transcoder-failure
   (is (true? @(client/create *c* K _set 1 100)))
   (let [transcoder (fn [_] (throw (Exception. "oh-no")))]
-    (is (thrown-with-msg? Exception #"oh-no" @(client/get-single *c* K _set {:transcoder transcoder})))))
+    (is (thrown-with-msg? Exception #"oh-no" @(client/get-single *c* K _set {:transcoder transcoder} [:all])))))
```

Comment:
all additions of `[:all]` to `get-single` are redundant now, right?

Reply:
you are correct! removed them all